andela/Titan-Devs
# 163518706 user able to highlight an article

#### What does this PR do?
Create CRUD functionality to highlight an article.
#### Description of Task to be completed?
- add create highlight endpoint
- add edit highlight endpoint 
- add update highlight endpoint 
- get all highlight endpoint
- get one highlight endpoint 
- add highlighted text to a comment if it has a highlight
#### How should this be manually tested?
*Clone the repo and check out ft-users-able-highlight-articles-163518706 branch
* Follow the instructions in ReadMe to setup the project
* Open Postman client
* Consume the API:
    * `post: /api/v1/article/:slug/highlight?start=3&end=5` supply the user 's token to create a highlight.
    * `put: api/v1/article/:slug/highlight?start=3&end=5` supply the user 's token to edit a highlight,  
    * `get /api/v1/article/:slug/highlight/:id` supply the user 's token to retrieve a given highlight.
    * `get /api/v1/article/:slug/highlights` supply the user 's token to get all highlights by a user on an article.
    * `/api/v1/article/:slug/highlight/:id` supply the user 's token to delete a highlight.
